github/github_test.go

```diff
@@ -1054,3 +1054,21 @@ func TestFormatRateReset(t *testing.T) {
 		t.Errorf("Format is wrong. got: %v, want: %v", got, want)
 	}
 }
+
+func TestNamedStructAccessor_noPanic(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/issues/1", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypeReactionsPreview)
+		fmt.Fprint(w, `{"number":1}`)
+	})
+
+	issue, _, _ := client.Issues.Get(context.Background(), "o", "r", 1)
```

Comment:
Let’s simplify this test. It’s not necessary to test `Issues.Get` method here, we already have dedicated tests for that. The scope of this test is to ensure the chained accessor methods on a nil do not panic only. So, you can replace all of the code above with just 1 line:

```Go
issue := &Issue{User: nil}
```

Reply:
Great! Will do that.